spec/project_config_spec.rb

```diff
@@ -927,10 +927,11 @@
                                                      "Variation mapped to experiment '#{@valid_experiment[:key]}' has been removed for user '#{@user_id}'.")
     end
     # Variation key is an empty string
-    it 'should delete forced varaition maping, log a message and return true when variation_key is passed as empty string' do
-      expect(config.set_forced_variation(@valid_experiment[:key], @user_id, '')).to eq(true)
+    it 'should return false when variation_key is passed as empty string' do
```

Comment:
should persist forced variation mapping, log a message and return false when variation_key is passed as empty string